aten/src/ATen/native/RNN.cpp

```diff
@@ -1194,6 +1194,53 @@ std::tuple<Tensor, Tensor, Tensor> quantized_lstm(
 
 }
 
+std::tuple<Tensor, Tensor, Tensor> quantized_lstm(
+      const Tensor& data, const Tensor& batch_sizes, TensorList hx,
+      TensorList _params, bool has_biases,
+      int64_t num_layers, double dropout_p, bool train, bool bidirectional,
+      c10::optional<ScalarType> dtype, bool use_dynamic) {
+  TORCH_CHECK(hx.size() == 2, "lstm expects two hidden states");
+  if (at::cudnn_is_acceptable(data)) {
+    Tensor output, hy, cy;
+    lstm_packed_cudnn_stub(data.type().device_type(), output, hy, cy, data, batch_sizes, hx,
+            _params, has_biases, num_layers, dropout_p, train, bidirectional);
+    return std::make_tuple(std::move(output), std::move(hy), std::move(cy));
+  }
+
+  if (use_miopen(data, dropout_p)) {
+    Tensor output, hy, cy;
+    lstm_packed_miopen_stub(data.type().device_type(), output, hy, cy, data, batch_sizes, hx,
+            _params, has_biases, num_layers, dropout_p, train, bidirectional);
+    return std::make_tuple(std::move(output), std::move(hy), std::move(cy));
+  }
```

Comment:
Do we need to add the same handler in Line 1160?

Reply:
Actually, not sure this should even be here. I just directly copy-pasted from the regular LSTM function

BTW do we actually support cudnn for these ops?

Reply:
Appears to be yes: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/miopen/RNN_miopen.cpp#L889

Reply:
@jianyuh I don't think those implementations (miopen and cudnn) actually support dynamic quantization

Reply:
Oh I thought you were asking if `lstm_packed_miopen` has cuda/cudnn implementation. Yep, the dynamic quantization is only supported on the CPU side.